test/TokenVesting.test.js

```diff
@@ -0,0 +1,171 @@
+const TokenVestingMock = artifacts.require('TokenVestingMock');
+const TokenMock = artifacts.require('TokenMock');
+import { latestTime } from 'openzeppelin-solidity/test/helpers/latestTime';
+import { increaseTimeTo, duration } from 'openzeppelin-solidity/test/helpers/increaseTime';
```

Comment:
There is a helper to get lastestTime, you can use that for test and `new Date().getTime()/1000` to get current unit tiemstamp instead the getNow funciton in contract

Reply:
I use GetNow() to be able to mock time in TokenVestingMock rather than to access time in tests.

Reply:
ok, you dont need it anyway, just use global var `now` and it will be the same